src/api/spec/models/bs_request_spec.rb

```diff
@@ -116,8 +116,8 @@
   end
 
   describe '#update_cache' do
-    RSpec.shared_examples 'bs_request after_commit callback' do
-      it 'touches the user' do
+    RSpec.shared_examples 'has associated object' do
```

Comment:
[Nitpick] "has associated object" doesn't really make sense to me, it seems like this shared context is more about making sure the users request cache is reset? Maybe say `it_should_behave_like "the user's cache is reset"`?